cmd/hiffy/src/lib.rs

```diff
@@ -111,49 +115,43 @@ pub enum HiffyLease<'a> {
     Write(&'a [u8]),
 }
 
-pub fn hiffy_list(hubris: &HubrisArchive, verbose: bool) -> Result<()> {
-    println!(
-        "{:<15} {:<12} {:<19} {:<15} {:<15}",
-        "TASK", "INTERFACE", "OPERATION", "ARG", "ARGTYPE"
-    );
-
+pub fn hiffy_list(hubris: &HubrisArchive, filter: Vec<String>) -> Result<()> {
     let print_args = |op: &(&String, &Operation), module, margin| {
         let mut args = op.1.args.iter();
-        let m = margin;
 
         match args.next() {
-            None => {
-                println!("-");
-            }
+            None => {}
             Some(arg) => {
-                println!("{:<15} {}", arg.0, arg.1.ty.0);
+                println!("{}{:<27} {}", margin, arg.0, arg.1.ty.0);
 
                 for arg in args {
-                    println!("{:m$}{:<15} {}", "", arg.0, arg.1.ty.0, m = m);
+                    println!("{}{:<27} {}", margin, arg.0, arg.1.ty.0);
                 }
             }
         }
 
-        if !verbose {
-            return;
-        }
-
         match idol::lookup_reply(hubris, module, op.0) {
-            Ok((_, Some(e))) => match &op.1.reply {
-                Reply::Result { ok, .. } => {
-                    println!("{:m$}{:<15} {}", "", "<ok>", ok.ty.0, m = m);
-                    println!("{:m$}{:<15} {}", "", "<error>", e.name, m = m);
-                }
-                _ => {
-                    warn!("Mismatch between expected reply and operation");
+            Ok((_, Some(e))) => {
+                match &op.1.reply {
+                    Reply::Result { ok, .. } => {
+                        println!("{}{:<27} {}", margin, "<ok>", ok.ty.0);
+                        println!("{}{:<27} {}", margin, "<error>", e.name);
+                    }
+                    _ => {
+                        warn!("mismatch on reply: expected Reply::Result, found {:?}",
+                    op);
```

Comment:
Nit - `cargo fmt` seems to have given up on this line. If I manually change this to

```rust
warn!(
    "mismatch on reply: expected Reply::Result, found {:?}",
    op
);
```

it then kicks back in and does some minor reformatting of the whole `match`.

Reply:
Ugh, thank you for spotting that!